filebeat/module/apache2/error/test/test.log-expected.json

```diff
@@ -1,36 +1,36 @@
 [
     {
         "@timestamp": "2016-12-26T16:22:08.000Z", 
-        "apache2.error.client": "192.168.33.1", 
-        "apache2.error.level": "error", 
-        "apache2.error.message": "File does not exist: /var/www/favicon.ico", 
         "event.dataset": "error", 
         "event.module": "apache2", 
         "input.type": "log", 
-        "log.offset": 0
+        "log.level": "error", 
+        "log.offset": 0, 
+        "message": "File does not exist: /var/www/favicon.ico", 
+        "source.ip": "192.168.33.1"
     }, 
     {
         "@timestamp": "2016-12-26T16:15:55.103Z", 
-        "apache2.error.level": "notice", 
-        "apache2.error.message": "AH00094: Command line: '/usr/local/Cellar/httpd24/2.4.23_2/bin/httpd'", 
         "apache2.error.module": "core", 
-        "apache2.error.pid": "11379", 
         "event.dataset": "error", 
         "event.module": "apache2", 
         "input.type": "log", 
-        "log.offset": 99
+        "log.level": "notice", 
+        "log.offset": 99, 
+        "message": "AH00094: Command line: '/usr/local/Cellar/httpd24/2.4.23_2/bin/httpd'", 
+        "process.pid": "11379"
     }, 
     {
         "@timestamp": "2011-09-09T10:42:29.902Z", 
-        "apache2.error.client": "72.15.99.187", 
-        "apache2.error.level": "error", 
-        "apache2.error.message": "File does not exist: /usr/local/apache2/htdocs/favicon.ico", 
         "apache2.error.module": "core", 
-        "apache2.error.pid": "35708", 
-        "apache2.error.tid": "4328636416", 
         "event.dataset": "error", 
         "event.module": "apache2", 
         "input.type": "log", 
-        "log.offset": 229
+        "log.level": "error", 
+        "log.offset": 229, 
+        "message": "File does not exist: /usr/local/apache2/htdocs/favicon.ico", 
+        "process.pid": "35708", 
+        "process.tid": "4328636416", 
```

Comment:
I wonder why this shows up as a string instead of a int here.

Reply:
Just hasn't been cast, I assume.

However I think the transient state of the JSON payload is not that critical:

1. The ECS template will define them as `long`, and ES will index them as such.
2. Re-reading them with Kibana will correctly indicate they are numeric
3. The only problem is that the `_source` will return this as a string, not a numeric.

Because of 3, we may want to fix this eventually. But I think it's out of scope for the mapping to ECS.

Reply:
@webmat Totally missed your comment here. I think you are actually spot on. I was kind of thinking if the type is defined as long it would also have an affect on _source but obviously it will not.